tidy3d/web/cli/app.py

```diff
@@ -114,5 +115,14 @@ def migration():
     migrate()
 
 
+@click.command()
+@click.argument("lsf_file")
```

Comment:
I think we can make use of some of click's features to handle the file stuff for us.
https://click.palletsprojects.com/en/8.1.x/arguments/#file-path-arguments